server/services/TransferService.js

```diff
@@ -128,12 +240,54 @@ class TransferService {
     try {
       await this._session.beginTransaction();
 
+      const transfer = await this._transfer.getById({
```

Comment:
Same thing here, would rather use the result of the declineTransfer function

Reply:
Confirmed this?

Reply:
confirmed at the top and changed all the payload where only transferId is saved to result